md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -402,14 +448,13 @@ RestWrapper export(@PathVariable("id") Integer processId
     }
 
     @RequestMapping(value = {"/zippedexport/{id}", "/zippedexport/{id}/"}, method = RequestMethod.GET)
-    @ResponseBody public
-    RestWrapper zippedexport(HttpServletResponse resp,
-                       @PathVariable("id") Integer processId
+    @ResponseBody
+    public RestWrapper zippedexport(HttpServletResponse resp,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "zippedexport" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)